extensions/wikia/YouTube/YouTube.php

```diff
@@ -457,26 +455,12 @@ function embedYouTube_url2tgid( $input ) {
 	return array( $tid, $gid );
 }
 
-function embedTangler( $input, $argv, $parser ) {
-	$tid = $gid = '';
-
-	if ( !empty( $argv['tid'] ) && !empty( $argv['gid'] ) ) {
-		list( $tid, $gid ) = embedYouTube_url2tgid( "{$argv['tid']}|{$argv['gid']}" );
-	} elseif ( !empty( $input ) ) {
-		list( $tid, $gid ) = embedYouTube_url2tgid( $input );
-	}
-
-	if ( !empty( $tid ) && !empty( $gid ) ) {
-		return "<p style=\"width: 410px; height: 480px\" id=\"tangler-embed-topic-{$tid}\"></p><script type=\"text/javascript\" src=\"http://www.tangler.com/widget/embedtopic.js?id={$tid}&gId={$gid}\"></script>";
-	}
-}
-
 function embedYouTube_url2gtid( $url ) {
 	$id = $url;
 
-	if ( preg_match( '/^http:\/\/www\.gametrailers\.com\/player\/(.+)\.html$/', $url, $preg ) ) {
+	if ( preg_match( '/^https?:\/\/www\.gametrailers\.com\/player\/(.+)\.html$/', $url, $preg ) ) {
 		$id = $preg[1];
-	} elseif ( preg_match( '/^http:\/\/www\.gametrailers\.com\/remote_wrap\.php\?mid=(.+)$/', $url, $preg ) ) {
+	} elseif ( preg_match( '/^https?:\/\/www\.gametrailers\.com\/remote_wrap\.php\?mid=(.+)$/', $url, $preg ) ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This branch's code block is the same as the block for the branch on line 461. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1871)

[project-id]: wikia:app